cigaradvisor/blocks/authorcard/authorcard.css

```diff
@@ -0,0 +1,224 @@
+.authorcard-container {
+    background: url('/cigaradvisor/images/author-card/media_17501a5f7bf5b4aebcf109765ead7cb5956480cbe.png');
+    margin-top: 50px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-color: transparent;
+    padding: 0 40px;
+    background-size: 900px 500px;
+    margin-bottom: 40px;
+}
+
+.authorcard-wrapper::after {
+    display: block;
+    clear: both;
+}
+
+.authorcard-wrapper {
+    display: flex;
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1080px;
+    align-items: stretch;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    align-content: flex-start;
+}
+
+.authorcard.block {
+    display: flex;
+}
+
+.authorcard.block .left-column,
+.right-column {
+    display: flex;
+    float: none;
+    width: 50%;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 10px;
+}
+
+.authorcard.block .left-column h2 {
+    font-family: "alfa slab one", cursive;
+    font-size: 40px !important;
+    color: #673841;
+    margin-top: 0;
+    padding-bottom: 10px;
+    border-image-source: url('/cigaradvisor/images/author-card/media_176afb26345b78f372ca811cf0e842c7ff75875a5.png');
+    border-image-slice: 50;
+    border-bottom: 30px solid;
+}
+
+.authorcard.block .left-column h2:first-child {
+    font-size: 35px !important;
+    border-bottom: none;
+}
+
+
+.authorcard.block .left-column p {
+    font-family: "open sans", sans-serif;
+    font-weight: 600;
+    color: #141414;
```

Comment:
We should use variables for colors. Something meaningful that can be changed if necessary.

--gold, --bronze, etc.

Reply:
fixed.